deeplabcut/utils/auxiliaryfunctions.py

```diff
@@ -115,17 +115,21 @@ def read_config(configname):
         try:
             with open(path, 'r') as f:
                 cfg = ruamelFile.load(f)
+                cfg['project_path'] = configname.replace('/config.yaml',
```

Comment:
Would that work on Windows? .replace(f'{os.path.sep}config.yaml', '') may be more robust.

Reply:
Hi Jessy!

That is definitely much better, I am applying the modifications